js-exercises/args-string/argsString.js

```diff
@@ -0,0 +1,16 @@
+const argsString = (message, args) => {
+  const messsageList = message.split('{}');
+  let argsIndex = 0;
+  let modifiedMessage = message;
+
+  for (const item of messsageList) {
```

Comment:
'message' can be used instead of  'item' if the array name is 'message list'.

Reply:
argsString already has message argument. To avoid confusion I used item.